ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -26,34 +26,71 @@
   retries: "{{ docker.pull.retries }}"
   delay: "{{ docker.pull.delay }}"
 
-- name: determine docker root dir
+- name: "determine docker root dir on docker-machine"
   uri:  url="http://{{ inventory_hostname }}:{{ docker.port }}/info" return_content=yes
-  register: dockerInfo
+  register: dockerInfo_output
+  when: whisk_version_name == "mac"
```

Comment:
whisk_version_name is only set to `mac` in ansible/environment/mac/group_vars/all